apps/ui/src/composables/useAiSummary.ts

```diff
@@ -0,0 +1,34 @@
+const SIDEKICK_URL = 'https://sh5.co';
+
+export function useAiSummary(proposalId: string) {
+  const aiSummary = ref<string>('');
+  const state = ref({
+    loading: false,
+    error: null,
+    open: false
```

Comment:
Personally I'd remove `open` from composable's state as I don't think it really belongs there. `open` is more of UI specific state and composable itself shouldn't care what UI does with it (it can just cover fetching summary and nothing more) - this value isn't managed by this composable as well.